packages/docs/src/routes/docs/(qwikcity)/guides/redirects/index.mdx

```diff
@@ -53,6 +53,23 @@ If you do not provide a status code, Qwik City will default to a `302` Found sta
 
 Read more about redirect status codes [here](https://developer.mozilla.org/en-US/docs/Web/HTTP/Status#redirection_messages).
 
+## Caching
+
+When you issue a redirect using the `redirect()` method from the `RequestEvent`, Qwik City applies a default cache control policy if one hasn't been explicitly set. This includes `Cache-Control` headers set in a higher-level layout or middleware. For more details on the order of execution, see the [caching documentation](/docs/caching/).
+
+If the redirect's status code is greater than `301` (e.g., `302`, `307`), and you haven't called `cacheControl()` for that request, the `Cache-Control` header will automatically be set to `'no-store'`. This prevents the redirect from being cached by the browser or intermediate caches.
+
+```typescript
+export const onGet: RequestHandler = ({ redirect, cacheControl }) => {
+  // This will result in a `Cache-Control: no-store` header except if cacheControl was called in a middleware or layout above.
+  throw redirect(302, '/new-location');
+
+  // To override the default, set it explicitly
+  cacheControl('day');
+  throw redirect(302, '/new-location');
+};
```

Comment:
Yeah looking at it like that on this code snippet, it really does look like a footgun and seems unintuitive. Will discuss this with the core team before merging.

Reply:
I would probably separate this code snippet into two code snippets each with its own explanation.